test/unit/test_verify.py

```diff
@@ -49,6 +54,15 @@ def test_files_contains(file1, file2, attributes, expect):
         files_contains(file2.path, file2.path, attributes)
 
 
+@pytest.mark.parametrize('file1,file2,attributes,expect', generate_tests())
```

Comment:
Use a separate function (or a flat list if you don't have to do anything fancy for generating the test case) for the second argument here (which should be an iterable that can be unpacked to as many values as you have arguments)